src/Arcus.WebApi.Tests.Unit/Security/Authorization/JwtTokenReaderTests.cs

```diff
@@ -7,24 +12,32 @@ namespace Arcus.WebApi.Tests.Unit.Security.Authorization
 {
     public class JwtTokenReaderTests
     {
+        private const string Issuer = "https://myut.com";
+        private const string Authority = "https://myut.com";
+        private const string PrivateKey = "J6k2eVCTXDp5b97u6gNH5GaaqHDxCmzz2wv3PRPFRsuW2UavK8LGPRauC4VSeaetKTMtVmVzAC8fh8Psvp8PFybEvpYnULHfRpM8TA2an7GFehrLLvawVJdSRqh2unCnWehhh2SJMMg5bktRRapA8EGSgQUV8TCafqdSEHNWnGXTjjsMEjUpaxcADDNZLSYPMyPSfp6qe5LMcd5S9bXH97KeeMGyZTS2U8gp3LGk2kH4J4F3fsytfpe9H9qKwgjb";
```

Comment:
I hope this is not sensitive information and just test value, but if it is test value, I'm wondering why it's hard-coded and not generated.

Reply:
done